byway/src/main/webapp/routepage.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="UTF-8">
+  <title>Route Generator</title>
+  <link rel="stylesheet" href="/bootstrap/bootstrap.min.css" crossorigin="anonymous">
```

Comment:
Note, you don't need crossorigin settings if you've saved bootstrap locally.

Reply:
Oh right, that's just for the CDN version. Removed.